Makefile

```diff
@@ -136,7 +147,6 @@ user_run:
 		--group-add sudo \
 		-v $(HOST_DIR):/host:z \
 		-v $(DOCKER_VOLUME_HOME):/home/$(shell whoami) \
-		-v $(ETC_LOCALTIME):/etc/localtime:ro \
```

Comment:
Had an issue with this on the M1 Mac, but I think those with a local computer rather than some VPS don't run into issues with this line -- need to check. Can't remember the error.

Reply:
Can confirm that this worked for me locally at least